From b2c39e8ca1bfb44ff951028eedb41446ed668ec8 Mon Sep 17 00:00:00 2001
From: Nikos Mavrogiannopoulos <nmav@gnutls.org>
Date: Tue, 9 May 2017 21:24:36 +0200
Subject: [PATCH] tests: modified pkcs1-pad to account for alt path search

Signed-off-by: Nikos Mavrogiannopoulos <nmav@gnutls.org>
---
 tests/cert-tests/pkcs1-pad | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gnutls28-3.4.10/tests/pkcs1-padding/pkcs1-pad
===================================================================
--- gnutls28-3.4.10.orig/tests/pkcs1-padding/pkcs1-pad
+++ gnutls28-3.4.10/tests/pkcs1-padding/pkcs1-pad
@@ -85,7 +85,7 @@ echo "PKCS1-PAD2 OK"
 # by Andrei Pyshkin, Erik Tews and Ralf-Philipp Weinmann.
 
 
-EXPECT3=02
+EXPECT3=03
 
 datefudge "2006-09-23" "${CERTTOOL}" --verify-chain --infile "${srcdir}/pkcs1-pad-broken3.pem" | tee out1 >/dev/null 2>&1
 
